src/sentry/api/endpoints/project_event_details.py

```diff
@@ -11,6 +11,50 @@
 from sentry.issues.query import apply_performance_conditions
 
 
+def wrap_event_response(event, event_data, project, requested_environments):
```

Comment:
Since we're passing the event here anyway does it make sense to just serialize insize this function?